docs/reference/search/search-your-data/search-across-clusters.asciidoc

```diff
@@ -290,14 +290,25 @@ means the document came from the local cluster.
 [[skip-unavailable-clusters]]
 === Skip unavailable clusters
 
-By default, a {ccs} returns an error if *any* cluster in the request is
-unavailable.
+By default, a {ccs} request fails if a remote cluster in the request is
+unavailable. Use the <<skip-unavailable,`skip_unavailable`>> cluster setting to
```

Comment:
if a remote cluster in the request returns an error?

Reply:
I changed this to:

> By default, a CCS fails if a remote cluster in the request returns an error or is unavailable.

I think it's still worth calling out an unavailable cluster as a specific case here. The behavior is different compared to other errors.